static/app/components/smartSearchBar/utils.tsx

```diff
@@ -351,6 +351,15 @@ export const shortcuts: Shortcut[] = [
   },
 ];
 
+const getItemTitle = (key: string, kind: FieldKind) => {
+  if (kind === FieldKind.FUNCTION) {
+    // Replace the function innards with ... for cleanliness
+    return key.replace(/(?<=\()[^\)]*/g, '...');
```

Comment:
[Is that a negative lookbehind?](https://caniuse.com/js-regexp-lookbehind)

I don't think you can use the lookbehind in safari 😢

Reply:
you're right

Reply:
I only know because I brought all of sentry down once with this

Reply:
oof